apps/openmw/mwgui/savegamedialog.cpp

```diff
@@ -402,32 +414,36 @@ namespace MWGui
         if (!mCurrentSlot)
             throw std::runtime_error("Can't find selected slot");
 
-        std::stringstream text;
+        std::stringstream savegameMetadata;
+
+        // Add real world datetime for last played.
         time_t time = mCurrentSlot->mTimeStamp;
         struct tm* timeinfo;
         timeinfo = localtime(&time);
 
-        text << std::put_time(timeinfo, "%Y.%m.%d %T") << "\n";
+        savegameMetadata << std::put_time(timeinfo, "%b%e, %Y %r %p") << "\n";
```

Comment:
interestingly, `%b` (abbreviated month name) seems to have a space after it, so no space is needed here between `%b` and `%e`